docs/creating_integration_file_structure.md

```diff
@@ -21,6 +21,11 @@ If your integration is going to integrate one or more devices, you will need to
 
 If your integration is going to register services, it will need to provide a description of the available services. The description is stored in `services.yaml`. [More information about `services.yaml`.](dev_101_services.md)
 
+## Data update coordinator - `coordinator.py`
+
+There are multiple ways for your integration to receive data, including push or poll. Commonly integrations will fetch data with a single coordinated poll across all entities, which requires the use of a `DataUpdateCoordinator`.
+If you want to use one, you'll have to create it in `coordinator.py`. [More information about `DataUpdateCoordinator`](integration_fetching_data.md/#coordinated-single-api-poll-for-data-for-all-entities).
```

Comment:
It's not required to subclass the DataUpdateCoordinator and it's not required to use a separate module. But we often recommend that.

Reply:
Thanks, made it a little more relaxed.